src/index.tsx

```diff
@@ -1,13 +1,31 @@
 import { createRoot } from 'react-dom/client';
-import { HashRouter as Router } from 'react-router-dom';
+import {
+  Navigate,
+  Route,
+  HashRouter as Router,
+  Routes,
+} from 'react-router-dom';
 
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { App } from './App';
+import { PeoplePage } from './components/PeoplePage';
+import { PeopleProvider } from './Context';
 
 createRoot(document.getElementById('root') as HTMLDivElement).render(
-  <Router>
-    <App />
-  </Router>,
+  <PeopleProvider>
+    <Router>
+      <Routes>
+        <Route path="/" element={<App />}>
+          <Route path="/home" element={<Navigate to="/" replace={true} />} />
+          <Route path="/people/:slug?" element={<PeoplePage />} />
```

Comment:
Ensure that the `<Route path='/people/:slug?' />` is correctly rendering the `PeoplePage` component. Verify that the `PeoplePage` component is correctly imported and used here.